cmd/security-spire-agent/Dockerfile

```diff
@@ -0,0 +1,61 @@
+#  ----------------------------------------------------------------------------------
+#  Copyright 2022 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+# 
+#  ----------------------------------------------------------------------------------
+
+# Build utility container
+ARG BUILDER_BASE=golang:1.17-alpine3.15
+FROM ${BUILDER_BASE} AS builder
+
+WORKDIR /edgex-go
+
+RUN sed -e 's/dl-cdn[.]alpinelinux.org/nl.alpinelinux.org/g' -i~ /etc/apk/repositories
+
+RUN apk add --update --no-cache make git
+
+COPY go.mod vendor* ./
+RUN [ ! -d "vendor" ] && go mod download all || echo "skipping..."
+
+COPY . .
+
+# The main mirrors are giving us timeout issues on builds periodically.
+# So we can try these.
+#RUN sed -e 's/dl-cdn[.]alpinelinux.org/nl.alpinelinux.org/g' -i~ /etc/apk/repositories
+#RUN apk add --update --no-cache zeromq-dev libsodium-dev pkgconfig build-base git
```

Comment:
Remove these and add the `sed` command into the alpine container below prior to fetching gcompat.